docs/intro/tutorial08.txt

```diff
@@ -0,0 +1,131 @@
+===================================
+
+A useful tool for debugging Django web applications is the Django Debug
+Toolbar. It's a third-party package maintained by the `Jazzband
+<https://jazzband.co>`_ organization. The toolbar helps you understand how your
+application functions and to identify problems. It does so by providing panels
+that provide debug information about the current request/response.
+
+- Cache panel: Presents you with the cache operations made and the context
+  of the executions.
+- Profiler panel: Presents you with a `cProfiler
+  <https://docs.python.org/3/library/profile.html>`_ summary of the codes
+  execution.
+- Request panel: Presents you with the headers and body of a request. This
+  allows you to determine what values are being sent to the Django application.
+- SQL panel: Presents you with all
+  the SQL queries made during the lifespan of a request.
+- Templates panel: Presents you with the templates used when rendering the
+  response and the context used to render them.
+
+To install a third-party application like the toolbar, you need to install
+the package by running the command within an activated virtual environment.
+This is like our earlier step of :ref:`installing an official release of
+Django <installing-official-release>`.
+
+   .. console::
+
+        $ python -m pip install django-debug-toolbar
+
+.. admonition:: Managing application dependencies
+
+    Managing dependencies is an important part developing an application, but
+    it is outside of the scope of this tutorial. Please see the official Python
+    documentation regarding `Managing Application Dependencies
+    <https://packaging.python.org/en/latest/tutorials/managing-dependencies/>`_
+    for more information.
+
+Third-party packages that integrate with Django may need some setup.
+Sometimes you will need to add the package's Django app to your
+:setting:`INSTALLED_APPS` setting. `The Django Debug Toolbar has many
+installation steps
+<https://django-debug-toolbar.readthedocs.io/en/latest/installation.html>`_.
+We're linking to the toolbar's documentation here because as a third-party
+package, it's maintained outside of the core development of Django. This
+allows the package maintainers to make releases outside of Django's release
+schedule
+
+Once installed, you should be able to see the DjDT "handle" on the right side
+of the browser window when you refresh the polls application.
+
+Getting help from others
+========================
+
+At some point you will run into a problem, for example the
+toolbar may not render. When this happens and you're unable to
+resolve the issue yourself, there are options available to you.
+
+#. If the problem is with a specific package, check if there's a
+   troubleshooting of FAQ in the package's documentation. For example the
+   Django Debug Toolbar has a `Tips section
+   <https://django-debug-toolbar.readthedocs.io/en/latest/tips.html>`_ that
+   outlines troubleshooting options.
```

Comment:
I think we should mention github issues for DjDT, even if only "search" rather than "report".

```suggestion
#. If the problem is with a specific package, check if there's a
   troubleshooting of FAQ in the package's documentation. For example the
   Django Debug Toolbar has a `Tips section
   <https://django-debug-toolbar.readthedocs.io/en/latest/tips.html>`_ that
   outlines troubleshooting options.
#. Search for similar issues on the package's issue tracker. Django Debug
   Toolbar’s is `on GitHub <https://github.com/jazzband/django-debug-toolbar/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc>`_.
```